src/components/ec-dropdown/ec-dropdown.spec.js

```diff
@@ -133,6 +136,11 @@ describe('EcDropdown', () => {
       const wrapper = mountDropdown({ items, isLoading: true });
       expect(wrapper.element).toMatchSnapshot();
     });
+
+    it('should render with a sensitive class when isSensitive prop is set to true when isSensitive prop is set to true', () => {
```

Comment:
duplicated spec text or (as I see in the component below) you wanted to mention about the nested `isSensitive`?